wasm_exec.js

```diff
@@ -189,12 +61,18 @@
     };
   }
 
+  // By removing the polyfills from this file we are more flexible in how we can use esbuild.
+  // These three globals are expected to be polyfilled elsewhere, for the specific build.
+  if (!global.crypto) {
+    throw new Error("must polyfill crypto.getRandomValues for tinygo wasm");
+  }
+
   if (!global.TextEncoder) {
-    global.TextEncoder = require("util").TextEncoder;
+    throw new Error("must polyfill util.TextEncoder for tinygo wasm");
```

Comment:
`TextEncoder` and `TextDecoder` are both now global in modern browsers and in node.js.
If we discover a need for a polyfill we can add one to the `polyfills` directory, but until then it seems good to just throw an Error.